README.md

```diff
@@ -107,6 +107,19 @@ scoop install openai-translator
 
 ## MacOS
 
+### Install via [brew cask](https://formulae.brew.sh/cask/)
+```bash
+brew install --cask openai-translator
+```
+
+> **_NOTE:_** If you don't have either brew or brew cask
+```bash
+# Install brew
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+# Install cask
+brew tap homebrew/cask
+```
+
```

Comment:
The below "Troubleshooting" section can be removed once the mac binaries are signed and notarized.